### PR TITLE
Remove unneeded top level dependencies

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,16 +11,14 @@
     "addons/storyshots/*",
     "app/*",
     "lib/*",
-    "examples/marko-cli",
-    "examples/angular-cli",
+    "examples/*-cli",
     "examples/cra-kitchen-sink",
     "examples/html-kitchen-sink",
```
Comment (pksunkara) [inline package.json]: Could have done `examples/*-kitchen-sink` too if not for `crna-kitchen-sink` package.

File: lib/cli/test/fixtures/riot/package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "riot": "^3.11.2"
   },
   "devDependencies": {
+    "cross-env": "^5.2.0",
```
Comment (Hypnosphi) [inline lib/cli/test/fixtures/riot/package.json]: Does it mean that user should install it before running `getstorybook`?

File: package.json [hunk 3]
```diff
@@ -92,8 +80,6 @@
     "concurrently": "^4.0.1",
     "cross-env": "^5.2.0",
     "danger": "^3.8.4",
-    "enzyme": "^3.5.0",
-    "enzyme-adapter-react-16": "^1.3.0",
```
Comment (Hypnosphi) [inline package.json]: why are those unneeded?

File: lib/cli/test/fixtures/riot/package.json (hunk 2 above)
Comment (pksunkara) [reply lib/cli/test/fixtures/riot/package.json]: Yes, see this package.json. It's something the user is supposed to consciously configure and use. Not something that should be a dependency of storybook package.

Comment (Hypnosphi) [reply lib/cli/test/fixtures/riot/package.json]: So storybook doesn't work without it? Why? We should fix or at least document it.

**Review by Hypnosphi — CHANGES_REQUESTED** [review]
https://github.com/storybooks/storybook/pull/4209#discussion_r220089348

File: lib/cli/test/fixtures/riot/package.json (hunk 2 above)
Comment (pksunkara) [reply lib/cli/test/fixtures/riot/package.json]: No, storybook works without it. I am saying this is something people use in their web apps which the fixture here is.

Comment (Hypnosphi) [reply lib/cli/test/fixtures/riot/package.json]: > No, storybook works without it

Let's not add it to the fixture without need then

Comment (pksunkara) [reply lib/cli/test/fixtures/riot/package.json]: `vue-cli` doesn't add it. I don't know why this fixture has it. You can see it here, https://github.com/storybooks/storybook/pull/4209/files#diff-a3134227a935db48acff355fe7fbedfdR8

**Review by Hypnosphi — APPROVED** [review]
Looks like that fixture actually uses cross-env

**Pull request merged**